pyaedt/modules/solutions.py

```diff
@@ -1088,18 +1091,21 @@ def ifft_to_file(
 
 
 class FfdSolutionData(object):
-    """Contains information from the far field solution data.
+    """Antenna array far-field data class.
```

Comment:
```suggestion
    """Provides antenna array far-field data.
```
A description of a class should start with a verb that ends in "s" or "es"